<launch>

  <node pkg="task3" type="only_movement.py" name="only_movement"  output="screen" required="true"/>

  <node pkg="task3" type="only_rings.py" name="only_rings"  output="screen"/>
  <node pkg="task3" type="only_clynders.py" name="only_clynders"  output="screen"/>
  <node pkg="task3" type="only_faces.py" name="only_faces"  output="screen"/>
  <node pkg="task3" type="only_parking.py" name="only_parking"  output="screen"/>

  
  <node pkg="task3" type="move_arm.py" name="move_arm"  output="screen"/>

  
  <node pkg="sound_play" type="soundplay_node.py" name="sound"  output="screen"/>
  
</launch>
